src/ansiblelint/__main__.py

```diff
@@ -244,7 +244,11 @@ def main(argv: list[str] | None = None) -> int:
     from ansiblelint.rules import RulesCollection
     from ansiblelint.runner import _get_matches
 
-    rules = RulesCollection(options.rulesdirs, profile_name=options.profile)
+    rules = RulesCollection(
```

Comment:
@ssbarnea Instead of sending three params we can just send one param `options` as the value for other params is just part of `options` object / dict